### DISCUSSION TITLE
Kibana Machine Learning

### DISCUSSION THREAD

**Discussion by LS3-Allen** [Security-Onion-Solutions/securityonion#4970] [Unsupported Versions]
Is it possible to enable Kibana Machine Learning?  I have found xpack.ml.enabled: false in elasticsearch.yml, but when I set it to true and restart elasticsearch it goes right back to false.  I'm assuming either minion.sls or global.sls needs to be modified.  Is that correct?

**Comment by lan94** [comment]
The most important thing is that you use the right license of Kibana/Elastic Stack. For Machine Learning you need at least [Platinum](https://www.elastic.co/pricing/).
